test/python/pulse/test_parameter_manager.py

```diff
@@ -14,491 +14,12 @@
 
 """Test cases for parameter manager."""
 
-from copy import deepcopy
-from unittest.mock import patch
-
-import numpy as np
-
 from qiskit import pulse
 from qiskit.circuit import Parameter
 from qiskit.pulse.exceptions import PulseError, UnassignedDurationError
-from qiskit.pulse.parameter_manager import ParameterGetter, ParameterSetter
-from qiskit.pulse.transforms import AlignEquispaced, AlignLeft, inline_subroutines
 from qiskit.test import QiskitTestCase
 
 
-class ParameterTestBase(QiskitTestCase):
-    """A base class for parameter manager unittest, providing test schedule."""
-
-    def setUp(self):
-        """Just some useful, reusable Parameters, constants, schedules."""
-        super().setUp()
-
-        self.amp1_1 = Parameter("amp1_1")
-        self.amp1_2 = Parameter("amp1_2")
-        self.amp2 = Parameter("amp2")
-        self.amp3 = Parameter("amp3")
-
-        self.dur1 = Parameter("dur1")
-        self.dur2 = Parameter("dur2")
-        self.dur3 = Parameter("dur3")
-
-        self.parametric_waveform1 = pulse.Gaussian(
-            duration=self.dur1, amp=self.amp1_1 + self.amp1_2, sigma=self.dur1 / 4
-        )
-
-        self.parametric_waveform2 = pulse.Gaussian(
-            duration=self.dur2, amp=self.amp2, sigma=self.dur2 / 5
-        )
-
-        self.parametric_waveform3 = pulse.Gaussian(
-            duration=self.dur3, amp=self.amp3, sigma=self.dur3 / 6
-        )
-
-        self.ch1 = Parameter("ch1")
-        self.ch2 = Parameter("ch2")
-        self.ch3 = Parameter("ch3")
-
-        self.d1 = pulse.DriveChannel(self.ch1)
-        self.d2 = pulse.DriveChannel(self.ch2)
-        self.d3 = pulse.DriveChannel(self.ch3)
-
-        self.phi1 = Parameter("phi1")
-        self.phi2 = Parameter("phi2")
-        self.phi3 = Parameter("phi3")
-
-        self.meas_dur = Parameter("meas_dur")
-        self.mem1 = Parameter("s1")
-        self.reg1 = Parameter("m1")
-
-        self.context_dur = Parameter("context_dur")
-
-        # schedule under test
-        subroutine = pulse.ScheduleBlock(alignment_context=AlignLeft())
-        subroutine += pulse.ShiftPhase(self.phi1, self.d1)
-        subroutine += pulse.Play(self.parametric_waveform1, self.d1)
-
-        sched = pulse.Schedule()
```

Comment:
You just need to remove this `sched` and directly insert 
```python
sched += pulse.ShiftPhase(self.phi3, self.d3)
```
into `long_schedule`, namely, in line 92
```python
long_schedule += pulse.ShiftPhase(self.phi3, self.d3)
```

This test checks if parameters can be assigned to `Schedule` wrapped by the `Call` instruction, as they have a bit tricky parameter management. Since `Call` instruction is removed with this PR, you don't need to test this. However, rest of program should be tested as before.